### ISSUE TITLE
Hover makes text jump on button

### ISSUE THREAD

**Issue by Graahf** [smashingmagazine/redesign#307]
Hi,

When looking at your memberships, I noticed that when hovering your "become a member" offer, the text on the button jumps.

I've illustrated the issue here
https://www.screencast.com/t/JjsW86b3D

I'm sitting on Chrome Version 62.0.3202.94 (Officiel version) (64-bit) - windows 10 (64bit)

Yours sincerly
Graahf

**- Do you want to request a *feature* or report a *bug*?**

**- What is the current behavior?**

**- If the current behavior is a bug, please provide the steps to reproduce.**

**- What is the expected behavior?**

**- Please mention your operating system version and the version of your browser**

**Comment by vitalyfriedman** [comment]
Dear @Graahf — is it possible to upload a recording in another format (not .swf), please?

**Comment by Graahf** [comment]
Hi,

Is this better?

https://drive.google.com/file/d/1kCX6qoWO6dZkjtnkfUxzs8G-hIHaAkjo/view

//Graahf

**Comment by vitalyfriedman** [comment]
Yes, it's much better! :-) Weird but I can't replicate this issue over here at all :-\ It looks like it's a rounding error, but it heavily depends on zooming level and the actual viewport. Does this issue occur on narrow screen and large screens as well? @Graahf

**Comment by Graahf** [comment]
Hmm, I just checked and it occurs on large screens (sitting with a viewport of 1920px x 1094px)

No issue on smaller screens
